src/GapicClientTrait.php

```diff
@@ -318,6 +318,20 @@ private function setClientOptions(array $options)
             $clientConfig,
             $options['disableRetries']
         );
+
+        // Edge case: If the client has the gRPC extension installed, but is
+        // a REST-only library, then the grpcVersion header should not be set.
+        if ($this->transport instanceof GrpcFallbackTransport) {
+            $options['grpcVersion'] = phpversion('grpc');
+            $options['restVersion'] = Version::getApiCoreVersion();
```

Comment:
As `GrpcFallbackTransport` doesn't actually use the `grpc` extension (or grpc), I'm not sure we want to set the `grpc/` header, as it's most likely going to be empty.

Reply:
Done.